backend/test_upload.sh

```
@@ -0,0 +1,49 @@
+# test_upload.sh
+#!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)